### PR TITLE
bulk version of getCollectionsForKeep

### PR DESCRIPTION
**Pull request by ymatsuda**
@eishay @andrewconner @ahsu1230 

Reducing the number of db calls and also using bulkGetOrElse to access the cache.

Martin is off the hook.

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/model/KeepToCollectionRepo.scala [hunk 1]
```diff
@@ -80,13 +81,34 @@ class KeepToCollectionRepoImpl @Inject() (
...
+        query.sortBy(_.updatedAt).map(_.collectionId).list // todo: we should add a column for explicit ordering of tags
       }
     } else {
       Seq.empty
     }
   }
 
+  def getCollectionsForKeeps(keeps: Seq[Keep])(implicit session: RSession): Seq[Seq[Id[Collection]]] = {
+
+    println("\n\t\t getCollectionsForKeeps\n")
```
Comment (ymatsuda) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepToCollectionRepo.scala]: oops, I will remove these line.

File: kifi-backend/modules/shoebox/app/com/keepit/model/KeepToCollectionRepo.scala [hunk 2]
```diff
@@ -68,7 +69,7 @@ class KeepToCollectionRepoImpl @Inject() (
         k <- keepRepo.rows if k.id === kc.bookmarkId && k.state === KeepStates.ACTIVE
       } yield kc
 
-      query.sortBy(_.updatedAt).map(_.collectionId).list // todo(martin): we should add a column for explicit ordering of tags
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepToCollectionRepo.scala:71]: lol :fishing_pole_and_fish: off the hook

**Comment by ahsu1230** [comment]
:+1: thanks!

**Comment by ymatsuda** [comment]
merging...

**Comment by eishay** [comment]
:-)

**Pull request merged**